frontend/apps/reader/modules/readerlink.lua

```diff
@@ -61,6 +61,10 @@ local function swipe_to_go_back()
     return G_reader_settings:readSetting("swipe_to_go_back") == true
 end
 
+local function swipe_to_follow_first_link()
```

Comment:
You mean function name should be swipeToFollowFirstLink() ?
grep'ing the full code base for `local function` shows 50% with _ , 50% with lowerUper, so I won't judge :)